BootloaderCommonPkg/BootloaderCommonPkg.dec

```diff
@@ -115,7 +115,13 @@
   gPlatformCommonLibTokenSpaceGuid.PcdSpiIasImage2RegionSize   |0x00000000|UINT32|0x2000019C
 
   ## This PCD controls enabled debug output devcie
-  gPlatformCommonLibTokenSpaceGuid.PcdDebugOutputDeviceMask    |0x00000003|UINT32|0x20000400
+  # Debug Interfaces. BIT0-RAM, BIT1-ISA UART, BIT2 -Not used, BIT3-USB3, BIT4-Serial IO, BIT5-TraceHub
+  gPlatformCommonLibTokenSpaceGuid.PcdDebugInterfaceFlags      |0x00000011|UINT32|0x20000400
```

Comment:
Why do we rename PCD and make changes to the bit definitions ?

Reply:
Mask seems to be more reasonable to me.  And it matches with other PCD such as ConsoleOuputDeviceMask, ConsoleInputDeviceMask, etc.

Reply:
PcdDebugInterfaceFlags is used in FSP UPD in 5 new platforms.  Moving PcdDebugInterfaceFlags from BootloaderCorePkg to BootloaderCommonPkg so that it could be used for all the drivers. PcdDebugInterfaceFlags could be used to set FSP UPD directly.  PcdDebugOutputDeviceMask is just removed to avoid duplicated.

Reply:
These FSP UPD is not architecture or common, so it might change at any time for different FSP.  While SBL common needs to be stable.  I think we should decouple from FSP UPDs, define what required from SBL design.  If required,  platform needs to do the remapping.

Reply:
Agree it is not necessary for SBL to change its definition to align with FSP.  SBL design should be able to provide required information for FSP. Currently PcdDebugOutputDeviceMask didn't provide if the serial port is ISA type or PCH type, the platform code need this info to set FSP UPDs.